pkg/cmd/cli/cmd/wrappers.go

```diff
@@ -213,7 +213,8 @@ const (
 
 // NewCmdPortForward is a wrapper for the Kubernetes cli port-forward command
 func NewCmdPortForward(fullName string, f *clientcmd.Factory) *cobra.Command {
-	cmd := kcmd.NewCmdPortForward(f.Factory)
+	// TODO: Make os.Stdout/Stderr configurable
+	cmd := kcmd.NewCmdPortForward(f.Factory, os.Stdout, os.Stderr)
```

Comment:
go ahead and plumb these through as args... there's only caller, and it already has out/errout, so there's minimal ripple